src/view/shell/index.tsx

```diff
@@ -154,6 +155,7 @@ function ShellInner() {
 
 export const Shell: React.FC = function ShellImpl() {
   const {fullyExpandedCount} = useDialogStateControlContext()
+  const lightStatusBarRefCount = useLightStatusBarRefCount()
```

Comment:
don't pass the ref count in context, pass the boolean. so that it only changes between `true` and `false` and not on individual ref count changes